tasks/agent.py

```diff
@@ -382,6 +396,8 @@ def hacky_dev_image_build(ctx, base_image=None, target_image="agent", push=False
 COPY --from=dlv /go/bin/dlv /usr/local/bin/dlv
 COPY --from=src /usr/src/datadog-agent {os.getcwd()}
 COPY --from=bin /opt/datadog-agent/bin/agent/agent /opt/datadog-agent/bin/agent/agent
+COPY dev/lib/libdatadog-agent-rtloader.so* /opt/datadog-agent/embedded/lib/
```

Comment:
Should we also copy `libdatadog-agent-three.so` and `libdatadog-agent-two.so` ?

Reply:
Indeed! Good catch!
Here is the change I did to also copy `libdatadog-agent-three.so`: https://github.com/DataDog/datadog-agent/pull/18375/commits/9b4b5571da2ed7a1d9aac9a5355084644dd2b55a